doc/api/fs.md

```diff
@@ -2217,4 +2217,4 @@ The following constants are meant for use with the [`fs.Stats`][] object's
 [`event ports`]: http://illumos.org/man/port_create
 [`ReadDirectoryChangesW`]: https://msdn.microsoft.com/en-us/library/windows/desktop/aa365465%28v=vs.85%29.aspx
 [`AHAFS`]: https://www.ibm.com/developerworks/aix/library/au-aix_event_infrastructure/
-[Common System Errors]: errors.html#errors_common_system_errors
\ No newline at end of file
+[Common System Errors]: errors.html#errors_common_system_errors
```

Comment:
Yea mine either. I noticed this same thing earlier. A newline seems to make the most sense.